projects/components/src/modal/modal.ts

```diff
@@ -8,6 +9,7 @@ export interface ModalConfig<TData = unknown> {
   title?: string;
   data?: TData;
   closeOnEscapeKey?: boolean;
+  styles?: Dictionary<unknown>;
```

Comment:
allowing arbitrary styles here could be bad. Can we just support only those customization that we need? Let's quickly discuss this on monday.

Reply:
@anandtiwary fixed it by defining an interface for style.